package.json

```diff
@@ -47,31 +49,30 @@
     "eslint-plugin-react": "^6.4.1",
     "eslint-plugin-standard": "^3.0.1",
     "jest": "^19.0.2",
-    "prettier": "^1.3.1",
-    "react": "^15.4.2",
-    "react-dom": "^15.5.0",
-    "react-test-renderer": "^15.5.0",
+    "prettier": "1.7.3",
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0",
+    "react-test-renderer": "^16.0.0",
     "semantic-release": "^6.3.6"
   },
   "peerDependencies": {
     "patternfly": "^3.27.4",
     "prop-types": "^15.5.10",
-    "react": "^15.5.0"
+    "react": "^15.6.2 || ^16.0.0",
+    "react-dom": "^15.6.2 || ^16.0.0"
   },
   "scripts": {
     "commit": "git-cz",
     "build": "babel src --out-dir lib --ignore test.js,__mocks__ ",
     "lint": "eslint --max-warnings 0 src storybook",
-    "prettier":
-      "prettier --write --single-quote --no-semi '{src,storybook}/**/*.js'",
+    "prettier": "prettier --write --single-quote --no-semi '{src,storybook}/**/*.js'",
```

Comment:
Prettier now supports prettierrc config file, so we can extract some of these CLI options into it.

Reply:
@jtomasek we can add this in a future PR along with prettier-stylelint if you want. I have added prettierignore to prevent conflicts in package.json

Reply:
fwiw, i still <3 Prettier... i did some testing recently and found that it resolves over 40% of the Standard JS Lint rules automatically ;)

https://gist.github.com/priley86/78093a1f37b39242096c13ddfbe3f701